stable/datadog/values.yaml

```diff
@@ -371,6 +371,24 @@ clusterAgent:
   #
   # readinessProbe:
 
+  ## @param service - object - optional
+  ##
+  # serviceType: ClusterIP
```

Comment:
This should be directly on top of `type: ClusterIP`, and prefixed with
`## @param type - string - optional`